recipes/swifter/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "swifter" %}
+{% set version = "0.286" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8703ed20105bda1017c7ed3254b62ceeb15e9f88c8c8d7cf2009c1a7edcc3a5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - dask >=0.19.0
```

Comment:
Are these all needed as host dependencies? Normally only `python` and `pip` are needed to run the `setup.py` script.

Reply:
I forgot to clean the recipe and put myself as the maintainer. I think everything is ok now. Thanks @chrisburr